<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="playerGroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Player Information</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter your name please</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameInput"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Enter your age please</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ageInput"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>How long have you watched SpongeBob? (select)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="watchCombo">
       <item>
        <property name="text">
         <string>My parents didnt let me watch it as a kid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Less than a year</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Over 2 years</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Over half my life</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>My whole life!</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start Quiz!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>410</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
    <property name="title">
     <string>SpongeBob Quiz</string>
    </property>
    <widget class="QLabel" name="questionLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>33</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>129</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Next Question</string>
     </property>
    </widget>
    <widget class="QLabel" name="questionNumberLabel">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
